docs/src/docs/versus.md

```diff
@@ -0,0 +1,88 @@
+# How KitOps is Different
+
+When people first come across KitOps they sometimes wonder, "how is this better than my favorite MLOps tool, git, Docker, etc...?"
+
+## How AI Project Assets are Managed Today
+
+Most teams working on AI projects store, track, and version their assets in one of two ways.
+
+1. Using an MLOps tool
+1. Using a combination of git, containers, cloud storage, and Jupyter notebooks
```

Comment:
```suggestion
2. Using a combination of git, containers, cloud storage, and Jupyter notebooks
```

Reply:
With Markdown you can just use `1.` for all the list items and when it's rendered it'll automatically present the correctly numbered list.

Reply:
yeah, i know, its a browser OL in the end :D , i just thought we wanted to keep it clean, but thats ok.